index.js

```diff
@@ -695,155 +695,96 @@ function loadTileSet(index) {
   }
 }
 
-function setTileSetOpacity() {
-  const tileSet = tileSets[currentIndex];
-  let imageIndex = 0;
-
-  // One loop per quadrant
-  for (let i = 0; i < tileSet.length; ++i) {
-    let imagesInQuadrant = [];
-
-    if (Array.isArray(tileSet[i])) {
-      // If it's a nested list, add all images to the same quadrant
-      imagesInQuadrant = tileSet[i];
-    } else {
-      // If it's a single image, treat it as its own group
-      imagesInQuadrant = [tileSet[i]];
-    }
-
-    // Determine which image should be visible in this quadrant
-    const visibleImageIndex = scrollIndex % imagesInQuadrant.length;
-
-    const opacityValue =
-      Number(document.getElementById(`opacityImage${i + 1}`).value) / 100; // Convert percent
-    const checked = document.getElementById(`image${i + 1}`).checked;
-    for (let j = 0; j < imagesInQuadrant.length; ++j) {
-      ++imageIndex;
-      if (checked) {
-        const tiledImage = viewer.world.getItemAt(imageIndex - 1);
-        if (tiledImage) {
-          if (j === visibleImageIndex) {
-            tiledImage.setOpacity(opacityValue);
-          } else {
-            tiledImage.setOpacity(0);
-          }
-        }
-      }
-    }
-  }
-}
-
 const viewerContainer = document.getElementById("viewer-container");
 
 // Share the mouse position between event handlers.
 let mousePos = new OpenSeadragon.Point(0, 0);
 
-// Divides the images at the current mouse position, clipping overlaid images to
+// Divides the images at the current mouse position, cropping overlaid images to
 // expose the images underneath. Note that all images are expected to have the
 // same position and size.
 const divideImages = () => {
-  const tileSet = tileSets[currentIndex];
   // Bail out if there are no images.
-  if (viewer.world.getItemCount() == 0) {
+  if (viewer.world.getItemCount() === 0) {
     return;
   }
 
-  // Get the clip point and clamp it to within the image bounds.
-  const image = viewer.world.getItemAt(0);
-  const clipPos = image.viewerElementToImageCoordinates(mousePos);
-  const size = image.getContentSize();
-  clipPos.x = Math.max(0, Math.min(clipPos.x, size.x));
-  clipPos.y = Math.max(0, Math.min(clipPos.y, size.y));
-
-  // Set the clip for each image.
-  let imageIndex = 0; // Because a given quadrant may have multiple images
-  let previousVisibleImages = 0;
-  // One loop per top level of image list
+  const tileSet = tileSets[currentIndex];
+  // Determine the number of checked images ahead of time so we know how big to
+  // make each section.
+  const isChecked = [];
+  let totalChecked = 0;
   for (let i = 0; i < tileSet.length; ++i) {
-    let imagesInQuadrant = [];
     const checkbox = document.getElementById(`image${i + 1}`);
+    isChecked.push(checkbox.checked);
+    if (checkbox.checked) {
+      ++totalChecked;
+    }
+  }
+
+  const radius = 4 * Math.max(window.innerWidth, window.innerHeight);

```

Comment:
This is not a tight fit, but I think 4 times the max window dimension is long enough that the edges of the sections should never be visible.